input/vocabulary/valueset-place-of-service.json

```diff
@@ -0,0 +1,510 @@
+{
+    "resourceType": "ValueSet",
+    "id": "place-of-service",
+    "meta": {
+        "profile": [
+            "http://hl7.org/fhir/StructureDefinition/shareablevalueset"
+        ]
+    },
+    "extension": [
+        {
+            "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-wg",
+            "valueCode": "pa"
+        },
+        {
+            "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status",
+            "valueCode": "draft"
+        },
+        {
+            "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm",
+            "valueInteger": 1
+        }
+    ],
+    "url": "https://signalbhn.org/fhir/ValueSet/place-of-service",
+    "version": "6.0.0",
+    "name": "PlaceOfService",
+    "title": "Place Of Service",
+    "publisher": "Signal",
+    "status": "draft",
+    "experimental": true,
+    "contact": [
+        {
+            "telecom": [
+                {
+                    "system": "url",
+                    "value": "http://hl7.org/fhir"
+                }
+            ]
+        }
+    ],
+    "description": "CMS maintains a list of Place of Service (POS) codes that indicate where a service can be provided.",
+    "compose": {
```

Comment:
No need to list out every code if you are including the entire code system.  Just use the include system, e.g.
```
  "compose" : {
    "include" : [{
      "system" : "http://terminology.hl7.org/CodeSystem/ex-serviceplace"
    }]
  }
```

Reply:
Just included the system to the valueset file. Kindly review @enjoysparkling